src/frontend/src/handler/create_sink.rs

```diff
@@ -342,6 +342,10 @@ pub async fn get_partition_compute_info(
 async fn get_partition_compute_info_for_iceberg(
     iceberg_config: &IcebergConfig,
 ) -> Result<Option<PartitionComputeInfo>> {
+    // TODO: check table if exists
+    if iceberg_config.create_table_if_not_exists {
```

Comment:
why can  do this?

Reply:
Because this is an optimization. We can always skip it. https://github.com/risingwavelabs/rfcs/blob/532d50c7f19ef940582390a17412e633e069449e/rfcs/0077-iceberg-sink-shuffle.md